test/common/http/filter/ip_tagging_filter_test.cc

```diff
@@ -0,0 +1,95 @@
+#include "common/buffer/buffer_impl.h"
+#include "common/http/filter/ip_tagging_filter.h"
+#include "common/http/header_map_impl.h"
+#include "common/http/headers.h"
+
+#include "test/mocks/http/mocks.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+using testing::_;
```

Comment:
I was following from the other test files. Seems that when we moved to the enclosing Envoy namespace all the usings got trapped inside.